### PR TITLE
Delivery for Distance and Direction Addin DotNet 2.4 - Sprint 1/1

### PR DESCRIPTION
**Pull request by saip**
This build addresses the GitHub tickets mentioned below:

- #608 - Distance Unit in Feature Tables is not based on Resource Value - [verified](https://github.com/Esri/distance-direction-addin-dotnet/issues/608#issuecomment-499670351)
- #635 - Cannot drag row to bottom in Range Rings Distances table - - [verified](https://github.com/Esri/distance-direction-addin-dotnet/issues/635#issuecomment-499861791)
- #639 - Add gradian as an angle unit of measure - [verified](https://github.com/Esri/distance-direction-addin-dotnet/issues/639#issuecomment-499871306)
- #647 - Clear and Save As buttons should have a hover action - [verified](https://github.com/Esri/distance-direction-addin-dotnet/issues/647#issuecomment-499872899)
- #648 - Using Addin with existing Pro project at a different scale causes "Spatial Index Grid Size" error - [verified](https://github.com/Esri/distance-direction-addin-dotnet/issues/648#issuecomment-500853894)

### CODE REVIEW

**Comment by csmoore** [comment]
Working with @saip we noticed some of the strange behavior I was seeing with D&D in Pro was caused by the lpkx not being package as "schema only" so I will repackage

**Comment by csmoore** [comment]
Once PR #662 is merged into this branch we can resume testing

**Comment by csmoore** [comment]
Verified that all issues except #648 are working as expected with this PR - with #648 we are experiencing a lockup issue on 2.4 daily as reported here: https://github.com/Esri/distance-direction-addin-dotnet/issues/648#issuecomment-499677301

File: source/addins/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProTabBaseViewModel.cs [hunk 1]
```diff
@@ -1349,6 +1351,23 @@ protected FeatureLayer GetFeatureLayerByNameInActiveView(string featureLayerName
             return viewLayer;
         }
 
+        internal static async void RemoveSpatialIndexOfLayer(string layerName)
```
Comment (csmoore) [inline source/addins/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProTabBaseViewModel.cs]: If this was a problem with the .lyrx layer package file having a spatial index - we could have run this tool prior to packaging the file to address this issue also

**Comment by csmoore** [comment]
I'm noticing the lockup problem https://github.com/Esri/distance-direction-addin-dotnet/issues/648#issuecomment-499677301 also in the dev branch so I feel confident it is not related to this PR. It is a very strange problem, that seems to require those exact set up steps. 

But at this point I think this PR is OK to merge if @saip has no more changes ( just noticed a commit so thought I would double check)

**Comment by saip** [comment]
@csmoore No more changes pending from us. Please merge.

**Pull request merged**